icrawler/downloader.py

```diff
@@ -5,6 +5,7 @@
 import threading
 
 import requests
+from urllib.parse import urlparse
```

Comment:
Sorry I thought I tried with python 2 and 3 but as I moved recently to python 3 my default python is 3 and no more 2. But in any case I didn't remember about `six`, thank you.